app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -42,6 +42,7 @@ def setup_form_object(step)
           begin
             form_name.constantize
           rescue NameError => x
```

Comment:
That's not the case. It's what I'd expect from previous rails projects, but for some reason missing a translation causes an error in the app.